addons/website/views/website_templates.xml

```diff
@@ -1714,6 +1714,7 @@
 </template>
 
 <!-- Footer templates -->
+<!-- Default -->
```

Comment:
Remove those things you added, they are useless: comments like this are only an occasion for typo or future dead code. The ID of the template literally contain the word you are commenting... and there is only one template below you comment.